src/resources/chats.jsx

```diff
@@ -0,0 +1,46 @@
+import { 
+    List, 
+    ReferenceField,
+    TextField, 
+    Datagrid,
+    Edit,
+    SimpleForm,
+    ReferenceInput,
+    TextInput,
+    Create,
+    DateTimeInput
+} from "react-admin";
+
+const chatFilters = [
+  <TextInput source="q" label="Search" alwaysOn />,
+];
+
+export const ChatList = () => (
+  <List filters={chatFilters}>
+    <Datagrid rowClick="edit">
```

Comment:
we always used "click to edit", but maybe we should also use the "show" page to display more information.

Reply:
i also thought about that but then, we display all the information in the Edit view so there is no more information we would display in show. I'm not sure if that's necessary then

Reply:
I think we should still have one and show an edit button or something at the end of the row. Otherwise it's pretty confusing if you just want to look at some data.

Reply:
yes okay maybe you are right. I'll add them